apple.china.conf

```diff
@@ -3,9 +3,11 @@ server=/a2.mzstatic.com/114.114.114.114
 server=/a3.mzstatic.com/114.114.114.114
 server=/a4.mzstatic.com/114.114.114.114
 server=/a5.mzstatic.com/114.114.114.114
+server=/aaplimg.com/114.114.114.114
```

Comment:
This rule seems too broad, what's the pattern under this domain? I would assume something like `itunes-apple.g.aaplimg.com`

Reply:
Two item is added for `aaplimg.com`.